src/MetadataManager/Metadata.ts

```diff
@@ -128,8 +125,7 @@ export class Metadata {
     }
 
     const filename: any = relPath.split('/').pop();
-    const stats: any =
-        await getStats(vscode.Uri.joinPath(vscode.workspace.workspaceFolders[0].uri, relPath));
+    const stats: any = fs.statSync(uri.fsPath);
```

Comment:
I changed this line and deleted `getStats` from Utils.ts.

Reply:
O.K. :D